bigtable-hbase-mirroring-client-1.x-parent/bigtable-hbase-mirroring-client-1.x/src/main/java/com/google/cloud/bigtable/mirroring/hbase1_x/bufferedmutator/SequentialMirroringBufferedMutator.java

```diff
@@ -202,6 +203,7 @@ private void addSecondaryMutation(
       reportWriteErrors(mutations, e);
 
       setInterruptedFlagInInterruptedException(e);
+      // TODO: use AccumulatedException instead.
```

Comment:
Just checking - is this supposed to be committed, or is it a reminder to yourself?

Reply:
i think that this comment is not longer present on the latest version of this branch. Ignoring.